app/controllers/miq_task_controller.rb

```diff
@@ -128,7 +128,11 @@ def cancel_task
   # Delete selected tasks
   def delete_tasks
     assert_privileges("miq_task_delete")
-    delete_tasks_from_table(find_checked_items, "Delete selected tasks")
+
+    checked_items = find_checked_items
+    new_checked_items = find_checked_items.select { |item| params["select-row-#{item}"] == "on" }
```

Comment:
The checked_items variable is never used. Did you mean:


```suggestion
    checked_items = find_checked_items
    new_checked_items = checked_items.select { |item| params["select-row-#{item}"] == "on" }
```